ansible/roles/matterbridge/tasks/main.yml

```diff
@@ -7,28 +7,28 @@
 
 - name: Template matterbridge config
   template:
-    src: matterbridge.toml.j2
+    src: "{{ inventory_dir | basename }}/matterbridge.toml.j2"
```

Comment:
@anadahz I would advocate against this. We are mixing up our own inventory structure with the roles. According to the docs, tho roles should be reusable and independent. If another organization chooses to use this role, they'll have a hard time figuring out why this is not working, I would say

Reply:
I'd suggest instead of doing it this way, to have variables on `matterbridge.toml.j2` that are overriden on the `testing` and `production` inventories as we usually do

Reply:
The file `matterbridge.toml.j2` for the testing host(s) should be only used for testing and nothing else. A separate template config of `matterbridge.toml.j2` should be created, otherwise the bridges may be mixed with the production and testing and this is something that we should avoid. 

Starting with a fresh `matterbridge.toml` should be the default and this is what the separate file enforces in order to avoid any issues while testing bridges.

Reply:
A simpler way without extra template directories will be to do use the `host_vars` depending on the inventory. I didn't use `group_vars` as `matterbridge.toml` must be unique per host so that we will not have any channel/user/bridge conflicts.

`src: "{{ inventory_dir }}/host_vars/{{ inventory_hostname }}/matterbridge.toml.j2"`

@francisco-core Does this looks good?

Reply:
This works for me but we should add a disclaimer for this either in a README in the role or having as the ask name `copying matterbridge config from "{{ inventory_dir }}/host_vars/{{ inventory_hostname }}/matterbridge.toml.j2"` such that people quickly know what the problem is.

Reply:
@francisco-core This is now fixed in: https://github.com/PrivacyLx/devops/pull/25/commits/0aab976b78df3d7d38ea250fcad3fa05bccec8ec